face_detector/launch/face_detector.narrow.launch

```diff
@@ -1,6 +1,5 @@
 <launch>
 
-  <include file="$(find face_detector)/launch/classifier.launch"/>
```

Comment:
This launch file hasn't existed (I even looked back at commits in 2009). The expected outcome of this feature will be added in the PR #30.